apps/web/components/Embed.tsx

```diff
@@ -809,67 +835,58 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
   ];
 
   return (
-    <DialogContent size="xl" className="p-0.5" type="creation">
+    <DialogContent size="lg" className="rounded-lg p-0.5" type="creation">
```

Comment:
```suggestion
    <DialogContent size="lg" className="rounded-lg p-0.5 enableOverflow" type="creation">
```

Reply:
Thank you for the review @CarinaWolli. You probably meant this?
```suggestion
    <DialogContent size="lg" className="rounded-lg p-0.5" enableOverflow type="creation">
```